rep-0152.rst

```diff
@@ -0,0 +1,147 @@
+REP: 152
+Title: Orphaned Pakcage Maintaining
+Author: Kei Okada
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 27-Dec-2017
+
+Outline
+=======
+
+#. Abstract_
+#. Motivation_
+#. Rationale_
+#. Operations_
+#. References_
+#. Copyright_
+
+Abstract
+========
+
+This REP specifies how we can organize and release packages without active maintainers.
+
+Motivation
+==========
+
+As the size of ROS ecosystem grows, there are several packages which loosing active maintainers.
+Main drawback of this situation is these packages do not get released into newer ROS distro coming every year, even if they are commonly used (For example, usb camera drivers are not released into Kinetic for more than half year [1]_ [2]_).
+Ideally everyone willing to use newer ROS distro almost equivalent to the set of packages in previous ROS distro.
+
+It is understandable that original maintainer could not continue serving as an active maintainer, because of starting new project, moving an institute/company, loosing interest and so on.
+And being a maintainer is not a easy job.
+It includes respond to incoming questions and issues, check for proposed changes and decide to include them to a main source tree and so on.
+
+The idea behind this document is to have a team of volunteers who can do ONLY the release job, which includes send source tree to ros build firm and act on notification mail form that.
+This will enable to continue using same set of previous ROS distros with minimum set of community efforts.
+To this end, we followed Debian QA Group System and Debian Developer's Reference ('Dealing with inactive and/or unreachable maintainers'[3]_), which is considered non-normative, but best practice.
+
+In ROS system, we use 'Unmaintained' package status([4]_) and ROS Orphaned Packages Group([5]_), who do not have to take full responsibility to package maintenance jobs, but expecting to take minimum responsibilities such as just to releasing into newer ROS distros.
+
+
+Rationale
+=========
+
+REP-0140 [6]_ and REP-0141 [7]_ provide maintainer information in package manifest file and maintainer status in ROS distribution file.
+
+Maintenance Guide in ROSWiki [8]_ provides how to claim maintainership for orphaned packages.
+
+Operations
+==========
+
+1-A. Orphaning a package by maintainers
+---------------------------------------
+
+If you can no longer maintain a your package.
+
+a. Set the package maintainer section in the ``package.xml`` file as follows.
+
+   ``<maintainer email="ros-orphaned-packages@googlegroups.com">ROS Orphaned Package Maintainers</maintainer>``
+
+b. Send email to ros-orphaned-packages@googlegroups.com and/or post to the `ROS Discourse Release Category <http://discourse.ros.org/c/release>`_, indicating that package is now orphaned.
```

Comment:
Have either a mailing list or (preferably) a Discourse category. We can subscribe to Discourse categories so there is no need for a separate mailing list. If a mailing list is needed to provide an email for the maintainer tag, then either find out if Discourse can provide that sort of feature or have just a mailing list rather than a Discourse category.